baxter/Dockerfile

```diff
@@ -15,9 +15,12 @@ WORKDIR   /home/ros
 RUN apt-get update && apt-get -y install software-properties-common libeigen3-dev
 RUN add-apt-repository ppa:ubuntu-toolchain-r/test
 RUN apt-get update && apt-get -y install gcc-4.9 g++-4.9
+RUN apt-get -y install gfortran libopenblas-dev liblapack-dev
```

Comment:
ok sorry last thing: can you please move this two lines up, ie before `add-apt-repository`? Lines 16-17-19-20 should all stay together because they pertain the installation and setup of `gcc-4.9`.